### PR TITLE
PS-8618 Apply new branding to docs (5.7)

### PR DESCRIPTION
**Pull request by patrickbirch**
new file:   docs/_static/Percona_Logo_Color.png
	new file:   docs/_static/percona-favicon.ico
	new file:   docs/_static/percona-favicon.png
	new file:   docs/_static/percona-favicon.svg
	modified:   docs/_static/percona-logo.svg
	modified:   docs/css/percona.css
	modified:   mkdocs-base.yml

### CODE REVIEW

File: mkdocs-base.yml [hunk 1]
```diff
@@ -18,6 +18,8 @@ theme:
   logo: _static/percona-logo.svg
   favicon: _static/percona_favicon.ico
```
Comment (alina-derkach-oaza) [inline mkdocs-base.yml]: wrong path. Change to _static/percona-favicon.ico

File: mkdocs-base.yml [hunk 2]
```diff
@@ -112,7 +114,16 @@ plugins:
 extra:
```
Comment (alina-derkach-oaza) [inline mkdocs-base.yml:114]: Update the path to cover_logo in "the with-pdf" section

cover_logo: docs/_static/Percona_Logo_Color.png

**Review by alina-derkach-oaza — COMMENTED** [review]
Remove the `header.html` file in `resources->overrides->partials`

File: mkdocs-base.yml (hunk 1 above)
Comment (patrickbirch) [reply mkdocs-base.yml]: done

File: mkdocs-base.yml (hunk 2 above)
Comment (patrickbirch) [reply mkdocs-base.yml:114]: Done

File: mkdocs-base.yml (hunk 1 above)
Comment (patrickbirch) [reply mkdocs-base.yml]: Done

**Review by alina-derkach-oaza — APPROVED** [review]
LGTM

**Pull request merged**